File: AnimGif/AnimGif/ViewController.m
//
//  ViewController.m
//  AnimGif
//
//  Created by Hiromasa OHNO on 2014/04/30.
//  Copyright (c) 2014年 xoyip. All rights reserved.
//

#import "ViewController.h"
#import <YLGIFImage.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad
{
    [super viewDidLoad];
	// Do any additional setup after loading the view, typically from a nib.
    
    self.imageView.image = [UIImage imageNamed:@"sample.gif"];
    self.gifImageView.image = [YLGIFImage imageNamed:@"sample.gif"];
}

- (void)didReceiveMemoryWarning
{
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: ObjcFromSwift/ObjcFromSwift/ViewController.swift
//
//  ViewController.swift
//  ObjcFromSwift
//
//  Created by Hiromasa OHNO on 2014/06/03.
//  Copyright (c) 2014年 xoyip. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
                            
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    @IBAction func start(sender : AnyObject) {
        SVProgressHUD.showWithStatus("loading...")
    }

    @IBAction func stop(sender : AnyObject) {
        SVProgressHUD.dismiss()
    }
}

